docs/admin/templates/managing-templates/devcontainers/index.md

```diff
@@ -0,0 +1,121 @@
+# Development containers
+
+A Development Container is an
+[open-source specification](https://containers.dev/implementors/spec/) for
+defining containerized development environments which are also called
+development containers (dev containers).
+
+Dev containers provide developers with increased autonomy and control over their
+Coder cloud development environments.
+
+By using dev containers, developers can customize their workspaces with tools
+pre-approved by platform teams in registries like JFrog Artifactory. This
+simplifies workflows, reduces the need for tickets and approvals, and promotes
+greater independence for developers.
+
+## Prerequisites
+
+An administrator should construct or choose a base image and create a template
+that includes a `devcontainer_builder` image before a developer team configures
+dev containers.
```

Comment:
We could link to the template example. Also, is it called `devcontainer_builder`?